param_templates/MOM_input.yaml

```diff
@@ -824,6 +833,7 @@ Global:
             HYBRID:vgrid.nc,sigma2,dz"
         datatype: string
         value:
+            $MOM6_VERTICAL_GRID == "zstar_65L": '"FILE:zstar_75layer_2.5m_248.4m-2024-03-29.nc,dz"'
```

Comment:
Should this be `zstar_75L`? `zstar_65L` is in the next block.

Reply:
Thanks for catching this, Mike. This has been fixed in the last commit.